config/storage.yml

```diff
@@ -0,0 +1,34 @@
+test:
```

Comment:
please rename this file to `config/storage.yml.example` and add `config/storage.yml` to git ignore